docs/docs/api/endpoints/components.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Components
+sidebar_label: Components
+---
+
+The `components.*` API is about controlling the individual components which are running inside `Airy Core`.
+
+## Get
+
+Retrieve the configuration of the components currently started in `Airy Core`.
+
+`POST /components.get`
+
+The list of the currently configured components is returned.
+
+**Sample response**
+
+```json5
+{
+  "security": {
+    "systemToken": "token",
```

Comment:
We want to make this API the new source of truth for interacting with Airy. Therefore I'd argue that as with all other endpoints we also convert these keys into `snake_case` so that the interface is uniform.